### PR TITLE
feat(normalize): add a bunch of tests from symfony/serializer component, to verify behavior

### PR DESCRIPTION
**Pull request by joelwurtz**

### CODE REVIEW

File: bench/composer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "automapper/bench",
```
Comment (Korbeil) [inline bench/composer.json:2]: I would prefer to split that in another PR please 🙏 (And maybe add a quick note about the output in documentation too ?)

Comment (joelwurtz) [reply bench/composer.json:2]: This is mainly for internal, to check some behavior and keeps them versioned, so we can link to them to why we do that in pull request or issue, not sure if we want that in doc ? could add a readme.md though

Comment (Korbeil) [reply bench/composer.json:2]: Let's add a readme then 👍

Comment (joelwurtz) [reply bench/composer.json:2]: done
